style.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h2 {
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 700;
+  max-width: 300px;
+  margin-bottom: 16px;
+  margin-top: 68px;
+  line-height: 52px;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: #172b4d;
+  margin-top: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to leave space on the top of the header in order to meet the provided Figma design.

    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/219373638-def5e849-8386-4dfe-8fe2-b0d54f2b1367.png)

    - ### FIGMA DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/219374023-53f05dd8-e602-4abb-a033-427b866c19a4.png)